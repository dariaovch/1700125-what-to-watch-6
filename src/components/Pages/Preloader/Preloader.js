import React from 'react';
// import spinner from 'src/images/Preloader.png';

function Preloader() {
  return (
    <div className="preloader">
      <img src="img/Preloader.png" alt="Спиннер поиска" className="preloader__spinner rotation" />
      <p className="preloader__text">Loading...</p>
    </div>
  );
}

export default Preloader;
