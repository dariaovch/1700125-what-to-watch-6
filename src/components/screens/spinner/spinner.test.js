import React from 'react';
import {render} from '@testing-library/react';
import Preloader from 'src/components/screens/spinner/spinner';

test(`Preloader component should render correctly`, () => {
  const {container} = render(<Preloader />);

  expect(container).toMatchSnapshot();
});
