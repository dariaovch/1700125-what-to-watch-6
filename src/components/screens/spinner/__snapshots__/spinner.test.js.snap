// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader component should render correctly 1`] = `
<div>
  <div
    class="preloader"
  >
    <img
      alt="Спиннер поиска"
      class="preloader__spinner rotation"
      src="img/Preloader.png"
    />
    <p
      class="preloader__text"
    >
      Loading...
    </p>
  </div>
</div>
`;
